### PR TITLE
feat: adding pipeline configuration document

### PR DESCRIPTION
**Pull request by tamirdavid1**

### CODE REVIEW

File: docs/pipeline/configuration.mdx [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+Odigos sets up an observability pipeline in your Kubernetes cluster to collect, process and export OpenTelemetry data.
+
+<Frame>
+    <img src="/images/odigos_pipeline.png" alt="Odigos Pipeline" />
+</Frame>
+
+
+User can configure the following components:
+
+- **Cluster Gateway Collector** - a collector that runs as a k8s Deployment. It receives the OpenTelemetry data from the Node Collectors, processes it, and exports it to the next component in the pipeline.
```
Comment (blumamir) [inline docs/pipeline/configuration.mdx:15]:
```suggestion
- **Cluster Gateway Collector** - a collector that runs as a k8s Deployment. It receives the OpenTelemetry data from the Node Collectors, processes it, and exports it to the configured destinations
```

File: docs/pipeline/configuration.mdx [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+
+### Configuring the Pipeline
+
+Odigos offers two main options for configuring the pipeline:
+<Info>
+When configuring the pipeline, you must use **only one of the two methods**—either profiles or directly editing the `odigos-config` ConfigMap. Using both methods will result in configurations overwriting each other, potentially leading to unexpected behavior.
```
Comment (blumamir) [inline docs/pipeline/configuration.mdx:21]: Can we check before applying the sizing profiles if the config exists and ignore / exit with error?

File: docs/pipeline/configuration.mdx [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+</Info>
+
+#### 1. Using Profiles
+
+Profiles are sets of pre-defined rules, specifically sizing profiles (`size_s`, `size_m`, `size_l`), to simplify the configuration of the pipeline. Each profile configures the following parameters for the **Cluster Gateway Collector**:
```
Comment (blumamir) [inline docs/pipeline/configuration.mdx:26]: I think it needs rephrasing

**Review by blumamir — APPROVED** [review]

File: docs/pipeline/configuration.mdx (hunk 2 above)
Comment (tamirdavid1) [reply docs/pipeline/configuration.mdx:21]: Will add this logic in another PR and their I will also change this message

**Pull request merged**